skuba-update/tox.ini

```diff
@@ -69,4 +69,4 @@ setenv =
 commands =
     flake8 --statistics -j auto --count {toxinidir}/skuba_update
     flake8 --statistics -j auto --count {toxinidir}/test
-    bash -c 'find . -name "*.sh" -exec shellcheck -e SC1090,SC1117 \{\} \+'
+    bash -c 'find {toxinidir}/test -name "*.sh" -exec shellcheck -e SC1090,SC1117 \{\} \+'
```

Comment:
Any idea why this is wrapped in a `bash -c` instead of just directly running find?

Reply:
I admit, I'm not familiar with 'shellcheck' as a command, so this whole line was a little confusing to me.

Reply:
Fixed both with `find {toxinidir}/test -name "*.sh" -exec shellcheck -e SC1090 \{\} +`
Thanks for your reviews.